api/udpa/data/orca/v1/orca_load_report.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package udpa.data.orca.v1;
+
+option java_outer_classname = "OrcaLoadReportProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.udpa.data.orca.v1";
+option go_package = "v1";
+
+import "validate/validate.proto";
+
+message OrcaLoadReport {
```

Comment:
Can you add a pointer and reference to the public design doc and relevant section?

Reply:
ACK.